erpnext/support/doctype/warranty_claim/warranty_claim.js

```diff
@@ -15,8 +15,6 @@ frappe.ui.form.on("Warranty Claim", {
 		frm.add_fetch('serial_no', 'maintenance_status', 'warranty_amc_status');
 		frm.add_fetch('serial_no', 'warranty_expiry_date', 'warranty_expiry_date');
 		frm.add_fetch('serial_no', 'amc_expiry_date', 'amc_expiry_date');
-		frm.add_fetch('serial_no', 'customer', 'customer');
```

Comment:
- Why change something in erpnext when you can always override it via custom app ??